src/containers/landing/Landing.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import MapPage from '../../components/map-page-components/map-page/MapPage';
+import LandingTreeStats from '../../components/landing-tree-status/LandingTreeStats';
+
+const Landing: React.FC = () => {
+  return (
+    <>
+      <Helmet>
```

Comment:
@jackblanc do we want Helmet on every page? I removed it because it keeps changing the tab text to things I don't want, just want it to say Speak for the Trees

Reply:
We want helmet on every page for search engine optimization. We don't have to include the title tag afaik. @willmt80 please update the name/content to generally reflect the content, doesn't have to be perfect yet.

Reply:
Should it just be on the Landing page or do we want Helmets on each component as well?

Reply:
I would say every container, not every component, but Jack might have a different idea